package.json

```diff
@@ -101,5 +101,9 @@
     "terra-toolkit": "^2.1.0",
     "webpack": "^3.6.0",
     "webpack-dev-server": "2.7.1"
+  },
```

Comment:
These should be removed

Reply:
Sorry. Forgot I added them here.

https://github.com/cerner/terra-core/pull/982/commits/a296bdf48a4acfbc9cb55ae29a66e87ec126b55e